update route format (#1632)
meilisearch/documentation: reference/api/keys.md

# Keys

> The `/keys` route allows you to create, manage, and delete API keys. In order to use these endpoints, the [master key must be set](/learn/configuration/instance_options.md#master-key) and supplied in the header of the request:

Reviewer: Just removing that "in order to" 😬 

```suggestion
The `/keys` route allows you to create, manage, and delete API keys. To use these endpoints, you [must set the master key](/learn/configuration/instance_options.md#master-key) and supply it in the header of the request:
```
